kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/index.vue

```diff
@@ -99,18 +134,27 @@
         selectResources,
         deselectResources,
         setSelectedResources,
+        notifyResourcesAdded,
+        notifySaveLessonError,
+        cancelAction$,
+        continueAction$,
         saveAndFinishAction$,
+        closeConfirmationTitle$,
+        closeConfirmationMessage$,
       };
     },
     data() {
       return {
         title: '',
         goBack: null,
+        isSaving: false,
+        isCloseConfirmationModalOpen: false,
         PageNames,
       };
     },
     computed: {
-      totalSize() {
+      ...mapState('lessonSummary', ['currentLesson', 'workingResources']),
```

Comment:
Oh, have we not migrated lesson editing over to using composition API for state? Are we updating the lessonSummary vuex whenever we edit things as well? cc @marcellamaki

Reply:
> Are we updating the lessonSummary vuex whenever we edit things as well?

Yes, thats why I had to call this `setWorkingResources` call [here](https://github.com/learningequality/kolibri/pull/12996/files/82cf6cf62f95ea7acc74ef6d839c6cdd336f92f2#diff-19b8c2b6b4d2b12a66a9d9a35c3a4e6b6312c432540bfbab30346aaa9ae83ea0R221).

And in the LessonSummaryPage, we also update this workingResources array  when we remove a resource from the lesson https://github.com/learningequality/kolibri/blob/0b448556530054c2b8c8a266a5d1281aa3c6461f/kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue#L343.